be/src/util/runtime_profile.cpp

```diff
@@ -45,7 +45,7 @@ static const std::string ROOT_COUNTER = "";
 
 RuntimeProfile::RuntimeProfile(const std::string& name, bool is_averaged_profile)
         : _pool(new ObjectPool()),
-          _own_pool(false),
+        //   _own_pool(false),
```

Comment:
just remove unused codes?

Reply:
https://user-images.githubusercontent.com/13197424/161288577-98d7382e-e476-4638-865a-9929bd5ffe68.png

Reply:
has been deleted by other PRs

Reply:
I means we just remove unused codes instead of comment them.

Reply:
Because I'm not sure if anyone else will use this later.
It's right to delete it.